mesonbuild/scripts/dist.py

```diff
@@ -62,6 +62,16 @@ def process_submodules(dirname):
     module_file = os.path.join(dirname, '.gitmodules')
     if not os.path.exists(module_file):
         return
+
+    # copy URLs from main configuration to avoid issues with relative paths
+    configs = subprocess.check_output(['git', 'config', '--get-regex', '^submodule\..*\.url$'],
+                                   stderr=subprocess.STDOUT).decode()
```

Comment:
__[Flake8]__ [E128] continuation line under-indented for visual indent

<sub>posted by Sider</sub>